pact/yaml/mainnet/ns.yaml

```diff
@@ -0,0 +1,29 @@
+codeFile: ../../ns.pact
+signers:
+  - public: 07b7f96e27d5c202c56a56a3e0dea569f2fa9c76274a1c9a19ecc970470011dc
+  - public: 1d062c51850de34ac86e47a70bcbdb68441b530074f8abb51978f6ac1717d5ef
+  - public: 56ff9e5c1b5ec3e98172a76aa3cdc03d3ec664f147267fd99eee4dd09763c853 #doug
+  - public: 7d08eec6c9e52dd39ec8cd29d71b2742aba033482d4a8da2aeeb5e14928f18d6
+  - public: a241430816f0112add2179e3e3939ebab5336ab024be990960e684e38eb87d80 #stu
+data:
+  ns: "kswap"
+  swap-ns-user:
+    keys:
+      - 8cc8da8ea7cbfaf6510dbc5f402b025dceaa9a3eaf7145e0ba933d468b63d358 #heekyun
+      - a241430816f0112add2179e3e3939ebab5336ab024be990960e684e38eb87d80 #stu
+      - 4b0f29b9e0a996587e5b5524731c91ecf02ecaa8b7e70e5f8f1881c5f0c18fc1 #francesco
+    pred: keys-any
+  swap-ns-admin:
+    keys:
+      - 8cc8da8ea7cbfaf6510dbc5f402b025dceaa9a3eaf7145e0ba933d468b63d358 #heekyun
+      - a241430816f0112add2179e3e3939ebab5336ab024be990960e684e38eb87d80 #stu
+      - 4b0f29b9e0a996587e5b5524731c91ecf02ecaa8b7e70e5f8f1881c5f0c18fc1 #francesco
+    pred: keys-any
+networkId: mainnet01
+nonce: install-kswap-namespace
+publicMeta:
+  chainId: "2"
+  sender: # add sender
```

Comment:
no opinion. I have a gas acct 9a4849687cbcfeb1f7c6510539638da576289508aedcc75f4d6ad3ed2623635c

Reply:
added in my acct `8cc8da8ea7cbfaf6510dbc5f402b025dceaa9a3eaf7145e0ba933d468b63d358`